functions/exercises.py

```diff
@@ -312,10 +355,13 @@ def reorder_hyphen_sequence(string):
     :param string:
     :return:
     """
-    # INSERT YOUR CODE HERE
-    #
-    # Run this command to test your implementation:
-    #
+    word = string.split('-')
+    word.sort()
+    string = word[0]
+    for i in range(1, len(word)):
+        string += f'-{word[i]}'
+
+    return string

```

Comment:
This could be simplified
```suggestion
    return '-'.join(word)
```